src/plugins/intel_cpu/src/transformations/snippets/aarch64/pass/brgemm_to_gemm_cpu.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "brgemm_to_gemm_cpu.hpp"
+
+#include "cpu_shape.h"
+#include "openvino/core/graph_util.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "snippets/itt.hpp"
+#include "snippets/op/brgemm.hpp"
+#include "snippets/op/buffer.hpp"
+#include "snippets/op/memory_access.hpp"
+#include "snippets/utils/utils.hpp"
+#include "transformations/snippets/aarch64/op/gemm_copy_b.hpp"
+#include "transformations/snippets/aarch64/op/gemm_cpu.hpp"
+#include "transformations/tpp/common/op/modifiers.hpp"
+#include "utils/general_utils.h"
+
+namespace ov::intel_cpu {
+
+using namespace snippets::lowered;
+
+namespace {
+template <typename T>
+void set_full_port_desc(const T& port) {
```

Comment:
The same helper is used in x64 brgemm_to_brgemm_cpu pass. Maybe we can move it to the utils to reuse it in both places? Also, we can use `enable_if` to use this method only for `ov::Output` and `ov::Input` `T` types

Reply:
moved to snippets utils for reuse. Thanks Vladislav!